backend/src/mail/constants.ts

```diff
@@ -38,7 +38,7 @@ export enum EmailSubjects {
   MEMBER_FOLLOW_UP = 'ACTION REQUIRED: Confirm or Decline Recommitment for the {{year}} CORE Program',
   MEMBER_NO_RESPONSE = 'CORE TEAM: Notice for Change in Member Status',
   MEMBER_ANNUAL = 'ACTION REQUIRED: Annual Recommitment Reminder. Confirm or Decline Recommitment for the {{year}} CORE Program',
-  MEMBER_REACTIVATE = 'ACTION REQUIRED: Update your CORE Recommitment status for {{year}}',
+  MEMBER_REACTIVATE = MEMBER_FOLLOW_UP,
```

Comment:
Hey I just wanted to make sure you tested this send? I was getting and error previously when using the same value here (which I assume is why you're setting it to the enum, clever, just wanted to double check :) )

Reply:
Yep, it seems to :)